### DISCUSSION TITLE
Stopping docker container leaves behind tmp_ recording files

### DISCUSSION THREAD

**Discussion by naanlizard** [OvenMediaLabs/OvenMediaEngine#1746] [Help]
When we deploy updates, we now use just `docker stop ome-container` after previous discussion about how that would now properly stop streams and process recordings - but it looks like that leaves behind tmp files in our ome recordings directory. 

Is this a bug, or are we doing something wrong? I would expect OME to gracefully shut down, including 

1. Disconnect incoming streams
2. process recordings
3. terminate process(es)

**Comment by naanlizard** [comment]
Also worth asking, how could we process these tmp files - just add the right extension? What are they?

**Reply by Keukhan** [reply]
If the recording is saved as .ts, adding the .ts extension to the tmp_ file can recover it. But recovery is not possible for .mp4 recordings

**Reply by Keukhan** [reply]
Please share the log file if the tmp_ file remains after stopping Docker. I will analyze it. :)

**Reply by Dawiducik** [reply]
I've successfully recovered "tmp" file that was left without .mp4 extension (suggesting something was wrong while finishing the file) with simple `Mp4 Video Repair Tool 1.0` application, cheers!
